src/quo2/components/share/qr_code/view.cljs

```diff
@@ -1,12 +1,76 @@
 (ns quo2.components.share.qr-code.view
-  (:require [quo2.components.share.qr-code.style :as style]
+  (:require [quo2.components.avatars.account-avatar.view :as account-avatar]
+            [quo2.components.avatars.channel-avatar.view :as channel-avatar]
+            [quo2.components.avatars.user-avatar.view :as user-avatar]
+            [quo2.components.avatars.wallet-user-avatar :as wallet-avatar]
+            [quo2.components.share.qr-code.style :as style]
             [react-native.core :as rn]
             [react-native.fast-image :as fast-image]))
 
-(defn qr-code
-  [{:keys [source width height]}]
-  [rn/view
-   {:style style/container}
+(defn- avatar-image
+  [{avatar-type :avatar
+    :as         props}]
+  [rn/view {:style style/avatar-overlay}
+   [rn/view
+    {:style (if (= avatar-type :wallet-account)
+              style/avatar-container-rounded
+              style/avatar-container-circular)}
+    (case avatar-type
+      :profile
+      [user-avatar/user-avatar
+       (assoc props
+              :size              :medium
+              :status-indicator? false
```

Comment:
are these props defaulted to `true`? weird if so 🫥

Reply:
yes o_o
![image](https://github.com/status-im/status-mobile/assets/90291778/06692b90-2c84-4254-8344-5b5f35b24062)